include/WCSimEnumerations.hh

```diff
@@ -15,12 +15,19 @@ typedef enum EDigitizerType {
   kDigitizerSKI
 } DigitizerType_t;
 
+typedef enum ERandomGeneratorType {
+  RANDOM_E_RANLUX=1,
+  RANDOM_E_RANECU=2,
+  RANDOM_E_HEPJAMES=3
+} WCSimRandomGenerator_t;
+
```

Comment:
This was done to keep `libWCSimRoot.so` compilable without a Geant4 installation; `libWCSimRoot.so` depends only on the files `WCSimRoot*.*`, `WCSimPmtInfo `WCSimEnumerations`